src/utils/apollo/paginateDepleteProjectsQuery.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  ProjectsDocument,
+  ProjectsQuery,
+  QueryProjectsArgs,
+} from 'generated/graphql'
+
+import client from './client'
+
+/**
+ * Loads all pages for the query.
+ * @param variables Variables to apply to query.
+ * @returns
+ */
+export async function paginateDepleteProjectsQueryCall({
```

Comment:
WDYT about maybe

```suggestion
export async function infiniteProjectsQuery({
```

Reply:
I am not sure how I feel about `infiniteProjectsQuery`. To me, when I read it, I think it is a pagination call, rather than depleting of the paginaition. `paginateDepleteProjectsQueryCall` is wordy but I know immediately what it is doing.

TBH, first thing that comes to mind to me when I see `infiniteProjectsQuery` is an infinity pool, so I think stream, rather than a depletion lol

Reply:
haha yea fair, tomato tomato. lfg